src/ig/IGExporter.ts

```diff
@@ -195,21 +196,64 @@ export class IGExporter {
   }
 
   /**
-   * SUSHI doesn't yet support authors adding additional pages beyond index.md.  If we detect
-   * additional pages, we need to log an error.
+   * Adds additional pages beyond index.md that are defined by the user.
+   * Only add formats that are supported by the IG template
+   * Intro and notes file contents are injected into relevant pages and should not be treated as their own page
+   *
+   * @param igPath {string} - the path where the IG is exported to
    */
-  private checkForOtherPageContent() {
+  private addOtherPageContent(igPath: string) {
     const inputPageContentPath = path.join(this.igDataPath, 'input', 'pagecontent');
     if (fs.existsSync(inputPageContentPath)) {
-      const pages = fs.readdirSync(inputPageContentPath);
-      if (pages.length > 1 || (pages.length === 1 && pages[0] !== 'index.md')) {
-        logger.error('SUSHI does not yet support custom pagecontent other than index.md.', {
+      const pages = fs
+        .readdirSync(inputPageContentPath)
+        .filter(page => page !== 'index.md')
+        .sort(); // Sorts alphabetically
+
+      let invalidFileTypeIncluded = false;
+      pages.forEach(page => {
+        // All user defined pages are included in input/pagecontent
+        const pagePath = path.join(this.igDataPath, 'input', 'pagecontent', page);
+        fs.copySync(pagePath, path.join(igPath, 'input', 'pagecontent', page));
+
+        const fileName = page.slice(0, page.lastIndexOf('.'));
+        const fileType = page.slice(page.lastIndexOf('.') + 1);
+        const isSupportedFileType = fileType === 'md' || fileType === 'xml';
+        const isIntroOrNotesFile = fileName.endsWith('-intro') || fileName.endsWith('-notes');
+        if (isSupportedFileType && !isIntroOrNotesFile) {
+          // If it is a valid file type and not an intro or note file, we will also add it to IG definition
+          this.ig.definition.page.page.push({
+            nameUrl: `${fileName}.html`,
+            title: `${fileName}`,
+            generation: fileType === 'md' ? 'markdown' : 'html'
+          });
+        } else {
+          invalidFileTypeIncluded = true;
```

Comment:
This looks to me like it's going to flag intro and notes files as invalid (because they will fall into this `else` clause).  I don't think that's what we want.

Reply:
Ah good point. I'll update.